params/config.go

```diff
@@ -451,6 +451,9 @@ type ChainConfig struct {
 
 	// Optimism config, nil if not active
 	Optimism *OptimismConfig `json:"optimism,omitempty"`
+
+	// Seconds per L2 block
+	BlockTime uint64 `json:"blockTime,omitempty"`
```

Comment:
It would be good to call this config var out as an op-stack diff with a comment

Reply:
Done in fed1b3c326f5ebb30783fdd33c58893bbeb82355.